crates/services/consensus_module/poa/src/pre_confirmation_signature_service.rs

```diff
@@ -41,6 +41,7 @@ pub mod tx_receiver;
 #[cfg(test)]
 pub mod tests;
 
+#[allow(dead_code)]
```

Comment:
```suggestion
```

Your local toolchain shouldn't affect the code that others are working on right now